manifests/reconfig_aem.pp

```diff
@@ -144,15 +146,54 @@
       }
     }
 
+    ###########################################################################
+    # If clean dir is enabled clean up directories in list
+    # If not enabled cleanup only any existing aem-healthcheck packages
+    ###########################################################################
+    if $enable_clean_directories {
+        $list_clean_directories = [
+        'install',
+        ]
+        #
+        # since we are only cleaning the install dir
+        # we clean during runtime.
+        #
+        $list_clean_directories.each | Integer $index, String $clean_directory| {
+          exec { "${aem_id}: Clean directory ${crx_quickstart_dir}/${clean_directory}/":
+            command => "rm -fr ${crx_quickstart_dir}/${clean_directory}/*",
+            before  => Exec["service aem-${aem_id} stop"]
+          } -> exec { "${aem_id}: sleep ${post_install_sleep_secs} seconds for package uninstallations":
+            command => "sleep ${post_install_sleep_secs}",
+          }
+        }
+    } else {
+      exec { "rm -f ${aem_base}/aem/${aem_id}/crx-quickstart/install/aem-healthcheck-content-*.zip":
+        before => [
+          Exec["service aem-${aem_id} stop"],
+        ]
+      }
+    }
+
     exec { "service aem-${aem_id} stop":
-      before  => Exec["service aem-${aem_id} start"],
+      before => [
+        Exec["service aem-${aem_id} start"],
+      ]
     }
 
-    exec { "service aem-${aem_id} start":
-      require => Exec["service aem-${aem_id} stop"],
+    aem_curator::install_aem_healthcheck {"${aem_id}: Install AEM Healthcheck":
```

Comment:
Would this cause any problem on a normal config_author or config_publish calls?
With the latest change following AEM 6.4 snapshot restructuring, the decision was to set the healthcheck package as part of machine image creation and reconfiguration from a (potentially non-aemopencloud) source repository.

Wouldn't this change replaces healthcheck on a normal config_author and config_publish as part of environment creation?